incubator/sparkoperator/values.yaml

```diff
@@ -26,6 +26,7 @@ metricsEndpoint: "/metrics"
 metricsPrefix: ""
 resyncInterval: 30
 webhookPort: 8080
+batchSchedulerName: ""
```

Comment:
Please add the code annotation for `batchSchedulerName `  here can make more sense.